src/php/php-fpm

```diff
@@ -0,0 +1,172 @@
+#!/bin/sh
+#
+# Startup script for PHP-FPM based on the directadmin.com script
+#
+# chkconfig: 345 85 15
+# description: PHP-FPM server
+# processname: php-fpm
+# config: $SNAP_DATA/php/php-fpm.conf
+
+PID=$SNAP_DATA/php/php-fpm.pid
+BIN=$SNAP/sbin/php-fpm
+CONFFOLDER=$SNAP
+FPMCONF=$CONFFOLDER/php/etc/php-fpm.conf
+CONFINI=$CONFFOLDER/php/lib/php.ini
+PHP_INI_SCAN_DIR=$SNAP/php/lib/php.conf.d
+export PHP_INI_SCAN_DIR
+OPTS="-R --daemonize --pid=${PID} --fpm-config $FPMCONF -c $CONFINI"
+
+#ulimit -HSn 32768
+
+# Setup PHP
+PHPFOLDER=$SNAP_DATA/php
+if [ ! -d "$PHPFOLDER" ]; then
+	echo "Configuring PHP-FPM..."
+	mkdir -p $PHPFOLDER
+	mkdir -p $PHPFOLDER/log
+	touch $PHPFOLDER/log/php-fpm.log
+	mkdir -p $PHPFOLDER/sockets
+fi
```

Comment:
That setup routine could be placed in a separate script, if better in terms of management.